CONTRIBUTING.md

```diff
@@ -231,6 +232,53 @@ To enabled Hot Reloading:
 
 Read more about [Hot Reloading](./docs/local-development.md#hot-reloading)
 
+### Logging
+
+Logging information can be very useful when developing, and there are a few logging options available to you.
+
+To enable logging:
+
+* [Create a local config file](#create-a-local-config-file) if you don't already have one
+* Edit your local config, changing the value of the logger type you want to see to `true`
+
+```json
+  "logging": {
+    "client": false,
+    "firefoxProxy": false,
+    "actions": true
+  }
+```
+
+Let's cover the logging types.
+
+#### client
+
+This option is currently unused.
+
+#### firefoxProxy
+
+This logger outputs a verbose output of all the Firefox protocol packets to your shell. Output looks like this.
+
+```
+WS -> TCP: 99:{"to":"server2.conn4.child5/consoleActor2","type":"startListeners","listeners":["NetworkActivity"]}
```

Comment:
Not sure we need to include examples of the output, it pretty verbose in the docs. These sections might be better as just this:

* `client` - This option is currently unused.
* `firefoxProxy` - Logs a verbose output of all the Firefox protocol packets.
* `actions` - Logs all redux actions.

I think there's something to be said for minimalism in things like this because it keeps the docs easily scannable. This document is already huge.